bin/dfx-ledger-get-icp

```diff
@@ -2,6 +2,15 @@
 set -euo pipefail
 SOURCE_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
 
+print_help() {
+  cat <<-EOF
+
+	Gives a user toy ICP, taken from an affluent friar
```

Comment:
Is the below key the authority on the faucet principal? Or where is it defined? ;-)

Reply:
Good point. :-) Added a reference to https://internetcomputer.org/docs/current/references/cli-reference/dfx-nns#_dfx_nns_install